src/Instrumentation/Psr18/composer.json

```diff
@@ -32,6 +32,13 @@
     "psalm/plugin-phpunit": "^0.16",
     "open-telemetry/sdk": "^1.0",
     "phpunit/phpunit": "^9.5",
-    "vimeo/psalm": "^4.0"
+    "vimeo/psalm": "^4.0",
+    "symfony/http-client": "^6.2",
+    "guzzlehttp/promises": "^1.5"
```

Comment:
Are these needed? I don't see any corresponding test changes that would require them...

Reply:
Nope, and I did not do this myself. I suspect it's the `php-http/discovery` plugin. Removed both changes for now